src/map/status.c

```diff
@@ -13397,25 +13397,127 @@ static bool status_readdb_sizefix(char *fields[], int columns, int current)
 	return true;
 }
 
-static bool status_readdb_scconfig(char *fields[], int columns, int current)
+static bool status_read_scdb_libconfig(void)
 {
-	int val = 0;
-	char* type = fields[0];
+	struct config_t status_conf;
+	const char *config_filename = "db/sc_config.conf"; // FIXME hardcoded name
```

Comment:
```suggestion
	const char *config_filename = map->db_path"/sc_config.conf"; // FIXME hardcoded name
```

Reply:
if do this fix, fixme comment can be removed